src/domains/profile/components/SignIn/SignIn.tsx

```diff
@@ -67,9 +69,10 @@ export const SignIn = ({ isOpen, profile, onCancel, onClose, onSuccess }: SignIn
 		}
 	}, [errors, remainingTime, setError, t]);
 
-	const handleSubmit = ({ password }: any) => {
+	const handleSubmit = async ({ password }: any) => {
 		if (profile.auth().verifyPassword(password)) {
```

Comment:
@faustbrian currently we are unable to verify the password when the user enters and submits the sign in form in order to proceed with the restoration. The `state` error is thrown. 

All the required data though is available in the profile and could be validated without the need to restore the profile first. Made https://github.com/ArkEcosystem/platform-sdk/pull/1389 as an approach for this, but happy to go with another one you think fits best.

Reply:
Will think about another solution as that breaks the single source of data for the authenticator instance because now any profile can use any authenticator instance of any profile which is nonsensical.

Reply:
https://github.com/ArkEcosystem/platform-sdk/commit/2056a696f3bdbf795c392b4a7c11435e1c724785

Reply:
@faustbrian looks great! Thanks